src/app/app-routing.module.ts

```diff
@@ -20,6 +29,18 @@ const routes: Routes = [
     data: {
       onAuthRequired
     }
+  },
+  {
+    path: 'student-login',
+    component: StudentHomePageComponent
```

Comment:
If you want these pages to be protected by login, you can use:
`canActivate: [OktaAuthGuard],` in the route. 

That way, you can't access those pages without logging in first. I'd suggest you add them before merging